content/en/agent/kubernetes/apm.md

```diff
@@ -17,25 +17,37 @@ To enable trace collection with your Agent, follow the instructions below:
 
 1. **Configure the Datadog Agent to accept traces**:
 
-    {{< tabs >}}
-{{% tab "Daemonset" %}}
+{{< tabs >}}
+{{% tab "Helm" %}}
 
-To enable APM trace collection in Kubernetes:
+**Note**: If you want to deploy the Datadog Agent as a deployment instead of a DaemonSet, configuration of APM via Helm is not supported.
 
-- Allow incoming data from port `8126` and set the `DD_APM_NON_LOCAL_TRAFFIC` and `DD_APM_ENABLED` variable to `true` in your *env* section of the `datadog.yaml` Agent manifest:
+- Update your datadog-values.yaml file with the following APM configuration:
```

Comment:
This should be preceded by a bullet point saying that the "Helm Installation" documented in your change above are a pre-requisite + a link to those steps. A lot of people will come to these page from google searches rather than by following a predetermined path through the documentation.